datasets/search_qa/search_qa.py

```diff
@@ -0,0 +1,222 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace NLP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""SEMPRE dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import logging
+import os
+
+import nlp
+
+_CITATION = """
+    @article{DBLP:journals/corr/DunnSHGCC17,
+    author    = {Matthew Dunn and
+                Levent Sagun and
+                Mike Higgins and
+                V. Ugur G{\"{u}}ney and
+                Volkan Cirik and
+                Kyunghyun Cho},
+    title     = {SearchQA: {A} New Q{\&}A Dataset Augmented with Context from a
+                Search Engine},
+    journal   = {CoRR},
+    volume    = {abs/1704.05179},
+    year      = {2017},
+    url       = {http://arxiv.org/abs/1704.05179},
+    archivePrefix = {arXiv},
+    eprint    = {1704.05179},
+    timestamp = {Mon, 13 Aug 2018 16:47:09 +0200},
+    biburl    = {https://dblp.org/rec/journals/corr/DunnSHGCC17.bib},
+    bibsource = {dblp computer science bibliography, https://dblp.org}
+    }
+
+"""
+
+_DESCRIPTION = """
+# pylint: disable=line-too-long
```

Comment:
Should this line be in the description?

Reply:
Oh no, better to move it outside. Thanks